.travis.yml

```diff
@@ -1,23 +1,35 @@
 language: ruby
 addons:
   ssh_known_hosts: github.com
+
+rvm:
+  - 2.5
+  - 2.6
+  - 2.7
+gemfile:
+  - Gemfile.rails52
+  - Gemfile.rails6
+  - Gemfile.rails61
+
```

Comment:
Just out of curiosity, how does the this naming convention work? Thanks!

Reply:
The naming convention here is just for ease of knowing what's in a file. As far as the Travis config is concerned, these are just three different Gemfiles; it's not interpreting anything special with the `.rails52` on the end. If you look down below, you'll see where the actual Rails version is being enforced within the file.